addons/assigngear/functions/fnc_gui_vehicleGear_selector.sqf

```diff
@@ -0,0 +1,222 @@
+#define DEBUG_MODE_FULL
+#include "\x\tmf\addons\assignGear\script_component.hpp"
+/*
+ * Name = TMF_assignGear_fnc_vehicleGear_selector
+ * Author = Head
+ *
+ * Description:
+ * Internal Use Only
+ */
+#include "\a3\3den\UI\dikCodes.inc"
+#include "\a3\3DEN\UI\resincl.inc"
+disableSerialization;
+params ["_mode", "_args"];
+TRACE_2("UI", _mode, _args);
+private _currentFilter = (uiNamespace getVariable [QGVAR(filter), FILTER_WEAPON]);
+private _data = (missionNamespace getVariable [QGVAR(vehicleGear_data), [ '', '', [ [],[],[] ] ]]);
+_data params [ ['_currentCategory', '', ['']], ['_currentFaction', '', ['']], ['_cache', [ [], [], [] ], [[]], 3] ];
+private _ctrlGroup = ctrlParentControlsGroup (_args # 0);
+private _fnc_getFactionItems = {
```

Comment:
This might as well be a separate function. Might be useful for other purposes.

Reply:
No